content/pt-br/observability.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Observabilidade
+status: Completed
+category: conceito
+tags: ["metodologia", "aplicação", "infraestrutura"]
+---
+
+## O que é
+
+Observabilidade é a capacidade de gerar e descobrir continuamente gatilhos acionáveis com base em sinais do sistema sob observação. 
```

Comment:
I feel like here we could keep `insights` instead of `gatilhos` wdyt @brunoguidone @edsoncelio @MrErlison ?

Reply:
I agree. We use the word `insights` a lot in Portuguese so I think can be better for this context.